<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorsWindow</class>
 <widget class="QDialog" name="DoctorsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>680</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="spacing">
    <number>9</number>
   </property>
   <item row="2" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>Historia:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../Resources/projectresources.qrc">:/images/Logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Widok Lekarski</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="2" column="0">
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Wyszukaj</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="patientList">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Pacjent:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="examination">
      <attribute name="title">
       <string>Przebieg badania</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="16" column="1">
        <widget class="QPushButton" name="patientClear">
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QPushButton" name="patientAccept">
         <property name="text">
          <string>Zatwierdź</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <layout class="QGridLayout" name="L4layout">
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Od</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QDateEdit" name="L4start">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2017</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Do</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QDateEdit" name="L4stop">
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2017</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QDateTimeEdit" name="examinationDate">
         <property name="dateTime">
          <datetime>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
           <year>2017</year>
           <month>1</month>
           <day>1</day>
          </datetime>
         </property>
         <property name="time">
          <time>
           <hour>8</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="generateL4cb">
         <property name="text">
          <string>Generuj L4</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Termin badania</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0" rowspan="5">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1" rowspan="10" colspan="4">
        <widget class="QTableWidget" name="symptomTable">
         <property name="minimumSize">
          <size>
           <width>320</width>
           <height>0</height>
          </size>
         </property>
         <property name="rowCount">
          <number>4</number>
         </property>
         <property name="columnCount">
          <number>2</number>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Objaw</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Opis</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="15" column="1" colspan="4">
        <widget class="QPlainTextEdit" name="diagnosis"/>
       </item>
       <item row="14" column="1" colspan="4">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Diagnoza</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Objawy</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1" colspan="4">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="addPatRow">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="removePatRow">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="receipt">
      <attribute name="title">
       <string>Tworzenie recepty</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="9" column="0">
        <widget class="QPushButton" name="prescriptionCreate">
         <property name="text">
          <string>Generuj</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Dodatkowe informacje</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTableWidget" name="drugs">
         <property name="rowCount">
          <number>4</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>100</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <column>
          <property name="text">
           <string>Lek</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Dawka</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Uwagi</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Leki</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="prescriptionClear">
         <property name="text">
          <string>Wyczyść</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPlainTextEdit" name="additionalInfo">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="removeMedRow">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="addMedRow">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources/projectresources.qrc"/>
 </resources>
 <connections/>
</ui>
